test/unit/test_tools_kraken.py

```diff
@@ -85,64 +85,3 @@ def test_classify_num_threads(self):
             self.assertIn('--threads', args)
             actual = args[args.index('--threads')+1]
             self.assertEqual(actual, str(expected), "failure for requested %s, expected %s, actual %s" % (requested, expected, actual))
-
-
-@unittest.skipIf(tools.is_osx(), "kraken osx binary does not yet exist on bioconda")
-class TestToolKrakenExecute(TestCaseWithTmp):
-
-    def setUp(self):
-        super().setUp()
-        input_dir = os.path.join(util.file.get_test_input_path(), 'TestMetagenomicsViralMix')
-        self.kraken_db_viral_mix = tempfile.mkdtemp('-kraken_db_viral_mix')
-        cmd = ['tar', '-C', self.kraken_db_viral_mix, '-xzf', os.path.join(input_dir, 'kraken_db.tar.gz')]
-        subprocess.check_call(cmd)
-        self.kraken = tools.kraken.Kraken()
-
-    def test_kraken_execution(self):
```

Comment:
Okay I think I understand all the bits and pieces of this PR now. The main change I want to see is that [`test_kraken`](https://github.com/broadinstitute/viral-ngs/blob/master/test/integration/test_kraken.py#L121) gain some of the output checking that is now lost by removing `test_kraken_execution` here. And I'm up for eliminating `test_kraken_tool`, `input_bam`, and `fastq_to_sam` from there as well to help clean it up. And if you want to work into this PR (or a future one) the testing of kraken-build, that'd be huge...